subsys/dfu/boot/targets/nrf52840_pca10056.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2017 Nordic Semiconductor ASA
+ *  Copyright (C) 2017, Linaro Ltd
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+#define FLASH_DRIVER_NAME		CONFIG_SOC_FLASH_NRF5_DEV_NAME
+#define FLASH_ALIGN			4
+#define FLASH_AREA_IMAGE_0_OFFSET	0x08000
```

Comment:
Same here. That you aren't getting redefinition warnings suggests that this device isn't in the device tree. The partitions should be added there, and not with these defines.